modules/twinkleconfig.js

```diff
@@ -19,7 +19,8 @@
  */
 
 
-Twinkle.config = {};
+Twinkle.config = twinkleconfigInit; // defined below
```

Comment:
I suppose it's silly to add the init bit to line 900+, but is there a good reason not to move the actual init function up here?  I say yes, having the objects that are 95% of what gets edited up top makes things easier for folks, especially new folks, but just wanted to mention.

Reply:
Yeah i think it's good to keep configurable data at the top of the file. It's a bit weird to take advantage of JS' variable hoisting but seems worth it.